bindata/network/openshift-sdn/002-rbac.yaml

```diff
@@ -31,6 +31,13 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups: [""]
+  resources:
+  - nodes
```

Comment:
you can just add `- nodes` to the existing `apiGroups: [""]` section above; it was separate before because it had different verbs than the other resources